src/Mapper/Mapper.test.ts

```diff
@@ -8,11 +8,13 @@ import { format, parse } from "date-fns";
 import { omit } from "lodash";
 
 import { RestSerializer } from "..";
-import { BaseService, DataService, registerService } from "../Services";
+import { BaseService, DataService, registerService, getDataService } from "../Services";
```

Comment:
`getDataService` and `MockMapper` (below) can be removed.

Reply:
ok... on it

Reply:
I also removed commented out code that we are not going to use.